Gemfile.jruby-2.5.lock.release

```diff
@@ -304,7 +303,7 @@ GEM
       logstash-core-plugin-api (>= 1.60, <= 2.99)
       rufus-scheduler
       stud (~> 0.0.22)
-    logstash-input-file (4.1.17)
+    logstash-input-file (4.1.18)
```

Comment:
Fix: sincedb_clean_after not being respected https://github.com/logstash-plugins/logstash-input-file/pull/276